sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -203,6 +205,13 @@ public EncryptedBlobAsyncClient buildEncryptedBlobAsyncClient() {
         Objects.requireNonNull(blobName, "'blobName' cannot be null.");
         checkValidEncryptionParameters();
 
+        if (EncryptionVersion.V1.equals(this.encryptionVersion)) {
+            LOGGER.warning("Client is being configured to use v1 of client side encryption, which is no longer "
+                + "considered secure. The default is v1 for compatibility reasons, but it is highly recommended "
+                + "the version be set to v2 using this builder");
+        }
+        this.encryptionVersion = encryptionVersion == null ? EncryptionVersion.V1 : encryptionVersion;
```

Comment:
should we make `EncryptedBlobClientBuilder` ctor take encryption version and deprecate existing one? This would make this more visible to user.

Reply:
Just saw that email. Yes I can change it.